.xo-config.json

```diff
@@ -10,6 +10,7 @@
 	"rules": {
 		"no-alert": "off",
 		"no-warning-comments": "off",
+		"unicorn/prefer-dom-node-dataset": "off",
```

Comment:
This rule breaks greppability because selectors in JS and CSS can't follow it. Change my view.

I still let it autofix our code where the attribute was not referenced anywhere else.